### PR TITLE
Added upload to album through website

### PR DESCRIPTION
**Pull request by Onestay**
Users are now able to upload to an album over the website. I'm not sure if the styling of the select box is okay with you but I found it pretty fitting.

### CODE REVIEW

File: public/js/home.js [hunk 1]
```diff
@@ -74,13 +103,13 @@ upload.prepareUpload = function(){
...
 	document.getElementById('uploadContainer').appendChild(div);
-
+	
 	upload.prepareDropzone();
 
 }
 
 upload.prepareDropzone = function(){
-
+
```
Comment (iCrawl) [inline public/js/home.js]: Please no trailing spaces kek

Comment (Onestay) [inline public/js/home.js]: oops

**Comment by Pitu** [comment]
This seems good, I'll test it locally tonight and let you know after. Thanks!

**Comment by Onestay** [comment]
👌

**Comment by Pitu** [comment]
@notstevek that's actually what I wanted to change before merging, could you submit it instead?

**Comment by notstevek** [comment]
@Pitu I would if it was coded, and not just me using gimp to edit it. I am not so great with css/html. Sorry for the confusion. :(

**Comment by Pitu** [comment]
@notstevek oh it's fine! Don't worry about it, I'll go ahead and make it then 🤗

**Comment by Onestay** [comment]
I thought of the styling like this as well. But the button and select were different sizes, they wouldn't line up and I hate CSS

**Pull request merged**